teuthology/kill.py

```diff
@@ -191,12 +194,12 @@ def find_pids(run_name):
     return run_pids
 
 
-def find_targets(run_name, owner):
+def find_targets(run_name, owner, job_id=None):
```

Comment:
```suggestion
def find_targets(owner, run_name, job_id=None):
```

Reply:
why do you want to change the order?

Reply:
because `run_name` is semantically closer to `job_id`. it's like pass `find_time_slot(user, year, month)`.

Reply:
good point, though let me not change the signature that much for now...